SPECS/ocaml-camlp-streams/ocaml-camlp-streams.spec

```diff
@@ -0,0 +1,110 @@
+# OCaml packages not built on i686 since OCaml 5 / Fedora 39.
+ExcludeArch: %{ix86}
+
+%ifnarch %{ocaml_native_compiler}
+%global debug_package %{nil}
```

Comment:
Why do we want to remove the debug package for some cases? It can be pretty useful.:) Please add a comment why we need to disable it.

Reply:
I am not sure, it seems we do not want it for some of the architectures. But in our case we build it on only 2